docs/validators/sigstore_cosign.md

```diff
@@ -92,8 +92,13 @@ kubectl run altsigned --image=docker.io/securesystemsengineering/testimage:co-si
 | `name` | - | :heavy_check_mark: | See [basics](../basics.md#validators). |
 | `type` | - | :heavy_check_mark: | `cosign`; the validator type must be set to `cosign`. |
 | `trustRoots[*].name` | - | :heavy_check_mark: | See [basics](../basics.md#validators). |
-| `trustRoots[*].key` | - | :heavy_check_mark: | See [basics](../basics.md#validators). ECDSA/RSA public key from `cosign.pub` file or [KMS URI](https://github.com/sigstore/cosign/blob/main/KMS.md). See additional notes [below](#kms-support). |
+| `trustRoots[*].key` | - | :heavy_check_mark: if not using keyless | See [basics](../basics.md#validators). ECDSA/RSA public key from `cosign.pub` file or [KMS URI](https://github.com/sigstore/cosign/blob/main/KMS.md). See additional notes [below](#kms-support). |
+| `trustRoots[*].keyless.Issuer` | - | :heavy_check_mark: if not using a key | Name of the OIDC provider which attests the identity. This is a regex, so either the full url of the OIDC provider can be used or only parts of it. |
+| `trustRoots[*].keyless.subject` | - | :heavy_check_mark: if not using a key | Name of the identity that created the keyless signature. Usually an email address. This is a regex, so either the full email address can be used or only parts of it (e.g. `your.mail@example.org` or `your.mail`). |
 | `host.rekor` | `rekor.sigstore.dev` | - | Rekor URL to use for validation against the transparency log (default sigstore instance is `rekor.sigstore.dev`). Setting `host` enforces successful transparency log check to pass verification. See additional notes [below](#transparency-log-verification). |
+| `host.rekorPubkey` | Public key of `rekor.sigstore.dev` | - | Public key used to verify signature of log entries from Rekor. If none is given the public key for `rekor.sigstore.dev` is being retrieved. |
+| `host.fulcioCert` | Root and intermediate certificates belonging to `fulcio.sigstore.dev` | - | The root certificate belonging the fulcio CA which was used to create keyless signatures. |
```

Comment:
Why certiface**s** in the default when the var name is singular?

Reply:
Because the value you actually enter is singular -> the root certificate.

Reply:
So the default is Root + intermediates, but a user could only specify root (which in practice would be the intermediate then)? Kinda weird, but I guess that works